src/js/fetch.js

```diff
@@ -0,0 +1,9 @@
+//funkcja odpowiada za pobranie danych następnie uruchamia callback
+const getData =(url,callback)=>{
+let data=fetch(url)
+.then(res=>res.json())
+//.then(res=>console.log(res))
+.then((res)=>{callback(res)})
+}
```

Comment:
Tu by można było poprawić wcięcia - kod byłby nieco bardziej czytelny. Można by też było zastanowić się nad dodaniem catcha.

Reply:
obsługi błędów wgl jeszcze nie zrobiłem, kod zaraz sformatuje bo masz racje że okropnie to wygląda